tests/e2e/e2e_tests/tests/containers.rs

```diff
@@ -0,0 +1,181 @@
+use std::path::PathBuf;
```

Comment:
> Asking because I don't know which one is a better option.

Basically, same. The problem is that `tests` folder already exists and it is related also for Go code tests.
I decided to go with the current approach -- small test crate inside `e2e` folder in mixed-language project.